### PR TITLE
add wxyz-* 0.4.1

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

This is an upstream monorepo that will (basically) always do synchronized releases that depend on each other, and will likely grow in size.

The `wxyz` proper package on pypi is a test package from two weeks ago (gah) that I didn't think to reserve.

Looking forward to exploring the complexity of updating this to JupyterLab 3, will need some kind of technique for extracting the licenses of bundled JS assets, I guess...

### CODE REVIEW

File: recipes/jupyter-wxyz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+        - pip
+        - python >=3.6
+      run:
+        - ipywidgets
+        - jsonpointer
+        - python >=3.6
+    test:
+      requires:
+        - pip
+        - jedi <0.18
```
Comment (bollwyvl) [inline recipes/jupyter-wxyz/meta.yaml:77]: Once https://github.com/conda-forge/jedi-feedstock/pull/32 is up on CDN, i'll drop all the jedi stuff...

**Comment by bollwyvl** [comment]
@conda-forge/staged-recipes @conda-forge/help-python ready for review!

**Review by beckermr — COMMENTED** [review]
Does this build produce a top-level output that is sensible?

**Comment by bollwyvl** [comment]
> Does this build produce a top-level output that is sensible?

Ehm, I have an atrocious `wxyz-notebooks` package that has `importnb`, `torch`, `lxml`, `skimage` and `tpot` and stuff in it... I don't really know if folk are going to want it! In my experience having deployed this a couple times, most projects don't end up wanting the kitchen sink, but I can certainly make one... though I might wait to do so until it supports JupyterLab 3, as the nodejs complexity is still rather high, as the js side is _also_ split up at the "heavyweight" dependency level... but really needed a Lab2-compatible build to shake out all the hiccups.

**Pull request merged**